loopchain/channel/channel_service.py

```diff
@@ -553,7 +553,9 @@ def score_write_precommit_state(self, block: Block):
         request = convert_params(request, ParamType.write_precommit_state)
 
         stub = StubCollection().icon_score_stubs[ChannelProperty().name]
-        stub.sync_task().write_precommit_state(request)
+        precommit_result: dict = stub.sync_task().write_precommit_state(request)
+        if precommit_result != hex(0):
```

Comment:
Since there are no public docs that shows the return value of icon-service score, `hex(0)` could raise a question. 

Please consider this:
```python
if "error" in precommit_result:
```